### PR TITLE
new mailchimp user testing signup form

### PR DESCRIPTION
**Pull request by SBBCornell**
A new docs page with a signup form that will add you to the mailchimp user testing group. See screenshot below. I hope we will migrate over from using the listserv in early January if all steps go smoothly.

screenshot-user-testing-signup

### CODE REVIEW

File: about/user-testing.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+<label for="mce-MMERGE6">Country * </label>
+<input type="text" value="" name="MMERGE6" class="required" id="mce-MMERGE6">
+</div>
+<div class="mc-field-group">
+	<label for="mce-RESEARCH">arXiv Research Field * </label>
+	<select name="RESEARCH" class="required" id="mce-RESEARCH">
+	<option value=""></option>
+	<option value="astro-ph">astro-ph</option>
+<option value="math">math</option>
+<option value="CoRR">CoRR</option>
```
Comment (jimentwood) [inline about/user-testing.md]: Please change CoRR to ...<option value="CS">Computer Science</option>

File: about/user-testing.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+</div>
+<div class="mc-field-group">
+<label for="mce-MMERGE6">Country * </label>
+<input type="text" value="" name="MMERGE6" class="required" id="mce-MMERGE6">
+</div>
+<div class="mc-field-group">
+	<label for="mce-RESEARCH">arXiv Research Field * </label>
+	<select name="RESEARCH" class="required" id="mce-RESEARCH">
+	<option value=""></option>
+	<option value="astro-ph">astro-ph</option>
```
Comment (jimentwood) [inline about/user-testing.md]: Please add other physics major groups
physics, cond-mat, hep

Comment (SBBCornell) [reply about/user-testing.md]: The list was getting a bit long so Ive instead updated the select options to match the Homepage main headings (Physics, Mathematics, etc). I think we only need broad strokes. @jimentwood does that sound reasonable?

Comment (jimentwood) [reply about/user-testing.md]: Yes, sounds good

**Pull request merged**